src/rules/typescript/extensions.ts

```diff
@@ -168,4 +168,41 @@ export = {
     // https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/space-infix-ops
     'space-infix-ops': 'off',
   },
+  overrides: [
```

Comment:
Could we move these overrides to a corresponding file? I usually prefer to put it tp the wrapper file, like `typescript.ts`.

Reply:
No, the rules files are collected in the specific wrapper, like `src/typescript.ts`. That's why I'm putting overrides in there.

Reply:
Maybe a separate file would make sense e.g. `overrides.ts` but I don't have a strong opinion.

Reply:
Sorry, my bad. I thought I put `overrides` in top-level files (that are supposed to be imported) but actually they are in rule files.